torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -70,7 +70,11 @@ bool isWeight(Module& module, Value* v) {
 
 Node* insertChooseQParams(Graph* graph, Value* original_val) {
   std::string choose_qparams_func = "_choose_qparams_per_tensor";
-  auto reduce_range = graph->insertConstant(false);
+  bool reduce_range_param = false;
+  if (at::globalContext().qEngine() == at::QEngine::FBGEMM) {
+    reduce_range_param = true;
+  }
+  auto reduce_range = graph->insertConstant(reduce_range_param);
```

Comment:
you probably forgot to insert this as an argument for some node?

Reply:
This is already being used in the choose_qparams node below.

Reply:
oh I didn't see the deleted line on the left